packages/core/review-workflows/server/src/services/assignees.ts

```diff
@@ -24,42 +24,49 @@ export default ({ strapi }: { strapi: Core.Strapi }) => {
      * Update the assignee of an entity
      */
     async updateEntityAssignee(
-      documentId: string,
-      locale: string,
+      entityToUpdate: {
+        id: number | string;
+        documentId: string;
+        locale: string;
+        updatedAt: string;
+      },
       model: UID.ContentType,
-      assigneeId: string
+      assigneeId: string | null
     ) {
-      if (isNil(assigneeId)) {
-        return this.deleteEntityAssignee(documentId, locale, model);
-      }
+      const { documentId, locale } = entityToUpdate;
 
-      const userExists = await getAdminService('user', { strapi }).exists({ id: assigneeId });
+      if (!isNil(assigneeId)) {
+        const userExists = await getAdminService('user', { strapi }).exists({ id: assigneeId });
 
-      if (!userExists) {
-        throw new ApplicationError(`Selected user does not exist`);
+        if (!userExists) {
+          throw new ApplicationError(`Selected user does not exist`);
+        }
       }
 
-      metrics.sendDidEditAssignee(await this.findEntityAssigneeId(documentId, model), assigneeId);
+      const oldAssigneeId = await this.findEntityAssigneeId(entityToUpdate.id, model);
```

Comment:
@remidej @markkaylor I will use this PR to also fix https://github.com/strapi/strapi/issues/20956 , we were sending the documentId to find the assignee id, but this method requires the entry id

Reply:
We dont have any assignee api test, so if thats fine by you I will create another pr with new review workflow tests so we can merge this one and we can do a hot fix (after the review :D )

Reply:
works for me 👍

Reply:
Sounds good 👍